style.css

```diff
@@ -148,10 +148,166 @@ body {
   color: #fff;
   border-radius: 4px;
   background: #0056EC;
+  border: 0;
+  transition: background .2s ease-out 0s;
+  outline: none;
 }
 
 .main__button:hover {
   cursor: pointer;
   background: #1e90ff;
-  transition: 1s;
+}
+
+.footer {
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  max-width: 80%;
+}
+
+.footer__circle {
+  position: relative;
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+  opacity: .7;
+  margin-right: 4px;
+  background: #c8dbfb;
+}
+
+.footer__circle:last-child {
+  margin-right: 0;
+}
+
+@media (max-width: 1366px) {
+  .wrapper {
+    max-width: 68%;
+  }
+
+  .header {
+    margin-bottom: 65px;
+  }
+
+  .menu__item {
+    font-size: 14px;
+    line-height: 16px;
+  }
+
+  .main__title {
+    padding-top: 40px;
+    font-size: 50px;
+    line-height: 66px;
+  }
+
+  .main__description {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 21px;
+  }
+
+  .main__button {
+    font-size: 14px;
+    line-height: 16px;
+  }
+
+  .main {
+    padding: 0 50% 50px 0;
+    background-size: contain;
+    margin-bottom: 90px;
+  }
+}
+
+@media (max-width: 1170px) {
+  .wrapper {
+    max-width: 91%;
+  }
+
+  .main__title {
+    margin-bottom: 34px;
+  }
+
+  .main__discription {
+    margin-bottom: 39px;
+  }
+}
+
+@media (max-width: 375px) {
+  .text {
+    display: none;
+  }
+
+  .menu {
+    position: absolute;
+    width: 25px;
+    height: 17px;
+    left: 326px;
+    top: 32px;
+  }
+
+  .menu-burger {
+    width: 25px;
+    height: 3px;
+    left: 326px;
+    background: #0056EC;
+  }
+
+  .menu-burger::after {
+    position: absolute;
+    display: block;
+    content: '';
+    width: 25px;
+    height: 3px;
+    top: 13px;
+    background: #0056EC;
+  }
+
+ .menu-burger::before {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tni-htmlcss2.0/pullRequest?prid=6206132)